### DISCUSSION TITLE
能否增加一个隐藏b站首页分区的功能

### DISCUSSION THREAD

**Discussion by cloud-wyf** [the1812/Bilibili-Evolved#4141] [功能建议 / Ideas]
![Snipaste_2023-04-11_22-31-29](https://user-images.githubusercontent.com/130480385/231967361-f3f628b0-c664-48d5-b7bc-d2212d8020e7.png)
如图，当b站首页下滑的时候，会出现如红圈所示分区，能否隐藏

**Comment by Geeyun-JY3** [comment]
内测首页是怎么整出图中这样没顶栏只有悬浮的分区栏的？

**Reply by cloud-wyf** [reply]
才发现我进的是内测的首页，内测首页下拉的话，默认顶部有两行，通过插件可以屏蔽第一行，但屏蔽不了第二行。其实直接退出内测就可以了
